puppet/modules/socorro/manifests/init.pp

```diff
@@ -62,11 +62,9 @@
   }
 
   package {
-    [
-      'apache-maven',
-    ]:
-    ensure  => latest,
-    require => Yumrepo['dchen'];
+    'apache-maven':
+      ensure  => latest,
+      require => Yumrepo['dchen'];
```

Comment:
Could do this as a separate bug, but we don't need this anymore (since we don't use hadoop/hbase)